### ISSUE TITLE
Question about CFG used in CogVideoX-5B-I2V

### ISSUE THREAD

**Issue by johnren-code** [THUDM/CogVideo#796]
When I finetune CogVideoX-5B-I2V in SFT, I find it use class free guidance for text condition in inference phase(default 5), but there is no text condition dropout during training phase in finetune code, may I ask is it a normal behaviour?
